### DISCUSSION TITLE
How do you import a project in a team account

### DISCUSSION THREAD

**Discussion by baptistemanson** [vercel/vercel#5043] [Help] [answered]
I've been trying several times over the last few months to import a project in a team account.

For some reason, the project always end up being imported in my personal account.
As it seems like too big to be a bug, what am I not understanding in the UX?

- I select my team account from the dropdown on the top left of the screen
- Then I click on import project
- I put the url of my project (I noticed that the url of the import project page drops the /dashboard/teamname, which may point towards a bug)
- I click import
- When done, I notice that the account on the top left corner is my personal account

UPDATE: My coworkers tried as well with no luck...So I'm not the only one confused right now at least!

**Comment by paulogdm** [comment, marked as answer]
Please double-check [this link](https://github.com/settings/installations) and remove the Vercel integration. It is possible that you integrated with your personal account instead of the team.

If you have any questions, feel free to DM me on twitter or use our [support form](https://vercel.com/support/request). We are also releasing improvements to reduce the friction of our integration.

Our sincere apologies for the poor experience.

**Comment by baptistemanson** [comment]
Hi Paulo.

You are right, this was the issues. You helped me on Saturday with
Guillermo.

Thanks!

<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail&utm_term=icon>
Virus-free.
www.avast.com
<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail&utm_term=link>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>

On Mon, Aug 17, 2020 at 5:42 PM paulogdm <notifications@github.com> wrote:

> Please double-check this link <https://github.com/settings/installations>
> and remove the Vercel integration. It is possible that you integrated with
> your personal account instead of the team.
>
> If you have any questions, feel free to DM me on twitter or use our support
> form <https://vercel.com/support/request>. We are also releasing
> improvements to reduce the friction of our integration.
>
> Our sincere apologies for the poor experience.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/vercel/vercel/discussions/5043#discussioncomment-52219>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABJRLAFLW6SANTAR2764YDDSBHE6TANCNFSM4P75YOZQ>
> .
>


-- 
Baptiste
